### ISSUE TITLE
Failed to activate the atom-typescript package

### ISSUE THREAD

**Issue by mailaneel** [atom/settings-view#495]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.187.0
**System**: Microsoft Windows 8.1 Enterprise
**Thrown From**: [settings-view](https://github.com/atom/settings-view) package, v0.184.0

### Stack Trace

Failed to activate the atom-typescript package

```
At Cannot read property 'bind' of undefined

TypeError: Cannot read property 'bind' of undefined
  at Package.module.exports.Package.activateServices (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\package.js:363:117)
  at Package.module.exports.Package.activateNow (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\package.js:243:16)
  at C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\package.js:221:30
  at Package.module.exports.Package.measure (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\package.js:167:15)
  at Package.module.exports.Package.activate (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\package.js:214:14)
  at PackageManager.module.exports.PackageManager.activatePackage (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\package-manager.js:465:21)
  at C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\node_modules\settings-view\lib\package-manager.js:226:29
  at exit (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\node_modules\settings-view\lib\package-manager.js:43:16)
  at BufferedProcess.triggerExitCallback (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\buffered-process.js:63:47)
  at ChildProcess.<anonymous> (C:\Users\ANiGUl\AppData\Local\atom\app-0.187.0\resources\app\src\buffered-process.js:85:18)
  at ChildProcess.emit (events.js:119:17)
  at Process.ChildProcess._handle.onexit (child_process.js:1072:12)

```

### Commands

```
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-typescript, v4.1.5
autocomplete-plus, v2.7.1
go-plus, v3.3.6
jshint, v1.3.1
linter, v0.12.1

# Dev
No dev packages
```

**Comment by lee-dohm** [comment]
You're on an older version of Atom. Please update to the latest version, v0.196.0, and let us know if this reproduces.

**Comment by thomasjo** [comment]
I believe this might be the dreaded module cache problem. I keep getting reports related to it on my packages whenever I update the main module's "ABI".

Sadly I'm failing right now at finding said issue for reference purposes.

**Comment by kevinsawicki** [comment]
I believe this was fixed by https://github.com/atom/atom/pull/6026 which checks for a function before calling bind on the service.

**Comment by kevinsawicki** [comment]
This was fixed in Atom 0.188
